triggers/service/src/main/resources/com/daml/lf/engine/trigger/db/migration/oracle/V1__Init.sql

```diff
@@ -0,0 +1,22 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Running trigger table.
+create table running_triggers(
+  -- a UUID generated by the service
+  trigger_instance nvarchar2(36) primary key,
+  -- the token corresponding to the party
+  trigger_party nvarchar2(255) not null,
+  -- the identifier for the trigger in its dalf,
+  --  of the form "packageId:moduleName:triggerName"
+  full_trigger_name nvarchar2(1000) not null
```

Comment:
1000 is somewhat arbitrary. There is no length limit on module name and trigger name. Oracle really does not like unlimited length (there are options but they all suck). The JSON API and canton also use somewhat arbitrary limits here.